dune/stuff/aliases.hh

```diff
@@ -6,17 +6,29 @@
 #ifndef DUNE_STUFF_ALIASES_HH
 #define DUNE_STUFF_ALIASES_HH
 
-namespace Dune {
-namespace Stuff {
+namespace Dune
+{
```

Comment:
I did not find a way to leave the `{` trailing in namespace declarations. Supposedly
```
BreakBeforeBraces: Mozilla
```
should do the trick but my clang 3.6.2. does not understand that (yet?).